### PR TITLE
feat(condo): DOMA-5402 create GetAccessTokenByUserIdService

### PR DESCRIPTION
**Pull request by VKislov**

### CODE REVIEW

File: apps/condo/bin/sbbol/crypto.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ const getAccessTokenFor = async (hashOrgId, userId) => {
     try {
         // `service_organization_hashOrgId` is a `userInfo.HashOrgId` from SBBOL, that used to obtain accessToken
         // for organization, that will be queried in SBBOL using `SbbolFintechApi`.
-        accessToken = await getAccessTokenForUser(userId)
+        [accessToken] = await getAccessTokenForUser(userId)
```
Comment (dkoviazin) [inline apps/condo/bin/sbbol/crypto.js:69]: if it is an array getAccessTokensForUser

File: apps/condo/domains/organization/integrations/sbbol/SbbolFintechApi.js [hunk 2]
```diff
@@ -213,7 +213,7 @@ const initSbbolFintechApi = async (userId) => {
     try {
         // `service_organization_hashOrgId` is a `userInfo.HashOrgId` from SBBOL, that corresponds to our organization
         // as a partner of SBBOL
-        accessToken = await getAccessTokenForUser(userId)
+        [accessToken] = await getAccessTokenForUser(userId)
```
Comment (dkoviazin) [inline apps/condo/domains/organization/integrations/sbbol/SbbolFintechApi.js:216]: getAccessTokensForUser

File: apps/condo/domains/organization/integrations/sbbol/SbbolSecretStorage.js [hunk 3]
```diff
@@ -41,7 +41,10 @@ class SbbolSecretStorage {
     }
 
     async getAccessToken (userId) {
-        return this.#getValue(`user:${userId}:accessToken`)
+        return [
```
Comment (dkoviazin) [inline apps/condo/domains/organization/integrations/sbbol/SbbolSecretStorage.js:44]: better to wrap it to object with keys

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/domains/organization/access/GetAccessTokenByUserIdService.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+const { SERVICE } = require('@condo/domains/user/constants/common')
+
+async function canGetAccessTokenByUserId (args) {
+    const { authentication: { item: user }, args: { data }, context } = args
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin) return true
+
+    if (user.type === SERVICE) {
+        const tokenAccessRights = await TokenAccessRight.getAll(context, {
```
Comment (pahaz) [inline apps/condo/domains/organization/access/GetAccessTokenByUserIdService.js:18]: getOne

**Pull request merged**